bin/test-workflow/policy/templates/app-identities-policy-jwt.template.yml

```diff
@@ -0,0 +1,26 @@
+---
+# This policy defines a group of whitelisted identities permitted to authenticate to the authn-jwt endpoint.
+- !policy
+  id: conjur/authn-jwt/{{ AUTHENTICATOR_ID }}/apps
+  annotations:
+    description: Identities permitted to authenticate
+  body:
+    - !group
+      annotations:
+        description: Group of authenticator identities permitted to call authn svc
+
+    - &hosts
+      - !host
+        id: system:serviceaccount:app-test:test-app-summon-sidecar
+        annotations:
+          authn-jwt/{{ AUTHENTICATOR_ID }}/kubernetes.io/namespace: {{ TEST_APP_NAMESPACE_NAME }}
```

Comment:
don't understand the point,
if the host id came from `sub` it already contains namespace, so why in this line namespace is a variable

Reply:
Good point  i replace in id